### ISSUE TITLE
Gather Vivado logs on CI failure

### ISSUE THREAD

**Issue by kgugala** [SymbiFlow/prjxray#785]
having the logs would really help in tacking the fuzzers instability issues

**Comment by litghost** [comment]
This command gather the all vivado logs:

```
cd fuzzers
find . -name vivado.log | xargs tar -czvf vivado.tgz
```

All of the Vivado logs compress into about ~1 MB, so kokoro should totally be able to store that :)

https://github.com/SymbiFlow/prjxray/blob/master/.github/kokoro/kokoro-cfg.py is used to determine what files to gather.

**Comment by litghost** [comment]
@tmichalak FYI

**Comment by kgugala** [comment]
I think this has been solved with #786
